sphinxcontrib/plantuml.py

```diff
@@ -422,6 +422,38 @@ def latex_depart_plantuml(self, node):
     pass
 
 
+def confluence_visit_plantuml(self, node):
+    fmt = self.builder.config.plantuml_output_format
+    if fmt == 'none':
+        raise nodes.SkipNode
+    try:
+        try:
+            fileformats, _ = _KNOWN_HTML_FORMATS[fmt]
+        except KeyError:
+            raise PlantUmlError(
+                'plantuml_output_format must be one of %s, but is %r'
+                % (', '.join(map(repr, _KNOWN_HTML_FORMATS)), fmt))
+        refname, outfname = render_plantuml(self, node, fileformats[0])
+    except PlantUmlError as err:
+        self.builder.warn(str(err))
+        raise nodes.SkipNode
```

Comment:
This looks like mostly a copy of `html_visit_plantuml()`. Can you extract a helper
function?

A separate commit is preferred.

Reply:
Sure, thanks for the suggestion @yuja